core/types/transaction_signing.go

```diff
@@ -71,7 +73,7 @@ func LatestSigner(config *params.ChainConfig) Signer {
 			return NewCancunSigner(config.ChainID)
 		}
 		if config.Cel2Time != nil {
-			return NewCel2Signer(config.ChainID)
+			return latestCeloSigner(config.ChainID, NewLondonSigner(config.ChainID))
```

Comment:
Mh, it doesn't solve the problem of ordering here.

Reply:
The problem is that we really need to get a hold of the signer returned by this function, but because it never puts the signer in a variable we have no way to capture it. I added an issue to get this changed upstream - https://github.com/celo-org/celo-blockchain-planning/issues/369